tx-source/dash_core.php

```diff
@@ -1073,3 +1073,54 @@
 $Definition['You cannot update this event\'s Zoom URL or ID.'] = 'You cannot update this event\'s Zoom URL or ID.';
 
 $Definition['Zoom Meeting ID or URL'] = 'Zoom Meeting ID or URL';
+
+// Automation Rules
+$Definition['A post was added to collection:'] = 'A post was added to collection:';
+$Definition['A post was inactive since:'] = 'A post was inactive since:';
+$Definition['A user has been registered for:'] = 'A user has been registered for:';
+$Definition['A user logs in or registers with email domain:'] = 'A user logs in or registers with email domain:';
+$Definition['A user registers or updates a profile field:'] = 'A user registers or updates a profile field:';
+$Definition['Add tag:'] = 'Add tag:';
+$Definition['Add to collection:'] = 'Add to collection:';
+$Definition['An idea with'] = 'An idea with';
+$Definition['and remove role'] = 'and remove role';
+$Definition['Assign role:'] = 'Assign role:';
+$Definition['Auto-run disabled.'] = 'Auto-run disabled.';
+$Definition['Auto-run enabled.'] = 'Auto-run enabled.';
+$Definition['Automation rule successfully created.'] = 'Automation rule successfully created.';
+$Definition['Automation rule successfully updated.'] = 'Automation rule successfully updated.';
+$Definition['Back'] = 'Back';
+$Definition['Bump the discussion'] = 'Bump the discussion';
+$Definition['Change the status of the idea to'] = 'Change the status of the idea to';
+$Definition['Close the discussion'] = 'Close the discussion';
+$Definition['Collapse'] = 'Collapse';
+$Definition['Create and manage automations to easily manage posts and users.'] = 'Create and manage automations to easily manage posts and users.';
+$Definition['Create/Edit Automation Rule'] = 'Create/Edit Automation Rule';
+$Definition['Edit Variables'] = 'Edit Variables';
+$Definition['Expand'] = 'Expand';
```

Comment:
Probably we already have a translation for this? If yes, no need to duplicate.

Reply:
I couldn't find an existing translation for "Expand".